zip-0317.rst

```diff
@@ -447,6 +515,14 @@ Possible alternatives for the parameters:
 of inputs/outputs to be non-proportional above the :math:`grace\_actions`
 threshold. This is no longer expressible with the formula specified above.)
 
+ZSA Fee Considerations
+======================
+
+We choose to maintain the native ZEC Asset as the primary token for the Zcash blockchain, similar to how ETH is needed for ERC20 transactions to the benefit of the Ethereum ecosystem.
+
+An alternative proposal for the ZSA fee mechanism that was not adopted was to adopt a new type of fee, denominated in the custom Asset being issued or transferred.
+In the context of transparent transactions, such a fee  allows to “tap into” the value of the transactions for the benefit of the miners. 
```

Comment:
There seems to be a missing word here.

Did you mean something like:
```suggestion
In the context of transparent transactions, such a fee allows miners to “tap into” the ZSA value of the transactions, rather than the ZEC value of transactions.
```